lib/collection.js

```diff
@@ -285,6 +285,7 @@ const DEPRECATED_FIND_OPTIONS = ['maxScan', 'fields', 'snapshot'];
  * @param {object} [options.collation] Specify collation (MongoDB 3.4 or higher) settings for update operation (see 3.4 documentation for available fields).
  * @param {ClientSession} [options.session] optional session to use for this operation
  * @param {boolean} [options.awaitData] Sets the cursor to block and await data for a while rather than returning no data
+ * @param {object} [options.readConcern] Specify a read concern for the operation. (only MongoDB 3.2 or higher supported)
```

Comment:
It would be nice, either for this PR or for another ticket, to create a jsdoc `@typedef` for a `readConcern`. That way, users of our documentation would be able to see exactly what goes into a `readConcern` object.

Reply:
I made NODE-1817 for this.